### PR TITLE
Design duplicate-alert & align game-create to middle

### PR DESCRIPTION
**Pull request by edwardycl**

### CODE REVIEW

File: frontend/src/app/lobby.component.ts [hunk 1]
```diff
@@ -72,6 +72,10 @@ export class LobbyComponent implements OnInit {
   }
 
   joinRoom(id: string) {
+    this.store.dispatch(new UserActions.RedirectWithSignInState({
```
Comment (tirr-c) [inline frontend/src/app/lobby.component.ts]: 서순을 바꾸자

**Comment by tirr-c** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit 8f64ace has been approved by `tirr-c`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit 8f64ace694fee1e38b5fca455e82c5ffb437f270 with merge d1d2ae306690a48aa01cd3498c6054aaafa24064...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/312260961?utm_source=github_status&utm_medium=notification)
Approved by: tirr-c
Pushing d1d2ae306690a48aa01cd3498c6054aaafa24064 to master...

**Pull request merged**